### PR TITLE
comment out the possibly test caused timeout

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy [hunk 1]
```diff
@@ -169,20 +170,20 @@ class APISpec extends Specification {
...
+//    def cleanup() {
+//        def options = new ListBlobContainersOptions().setPrefix(containerPrefix + testName)
+//        for (BlobContainerItem container : primaryBlobServiceClient.listBlobContainers(options, Duration.ofSeconds(120))) {
+//            BlobContainerClient containerClient = primaryBlobServiceClient.getBlobContainerClient(container.getName())
+//
+//            if (container.getProperties().getLeaseState() == LeaseStateType.LEASED) {
+//                createLeaseClient(containerClient).breakLeaseWithResponse(0, null, null, null)
+//            }
+//
+//            containerClient.delete()
```
Comment (hemanttanwar) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: Wondering how these resources will be cleaned up  after test are run ?

Comment (sima-zhu) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: The PR is for testing the live tests for storage. It is not for real issue.

**Comment by sima-zhu** [comment]
Close the one and move to alan's PR

**Pull request closed without merging**